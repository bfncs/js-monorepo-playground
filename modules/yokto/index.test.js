import React from 'react';
import renderer from 'react-test-renderer';
import Yokto from './';

const NO_OP = () => {};

describe('react.input snapshot tests', () => {

  it('matches snapshot when done', () => {
    expect(renderer.create(
      <Yokto id={'someId'}>Some content</Yokto>
    ).toJSON()).toMatchSnapshot();
  });

});