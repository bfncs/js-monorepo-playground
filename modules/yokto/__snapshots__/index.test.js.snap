// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react.input snapshot tests matches snapshot when done 1`] = `
<div>
  Yokto: 
  Some content
  <br />
  id: 
  someId
</div>
`;
